threeplay.go

```diff
@@ -25,17 +25,11 @@ const (
 	HTML OutputFormat = "html"
 )
 
-// HTTPClient interface for http clients used in the 3Play client
-type HTTPClient interface {
-	Get(string) (*http.Response, error)
-	PostForm(string, url.Values) (*http.Response, error)
-}
-
 // Client 3Play Media API client
 type Client struct {
-	apiKey    string
-	apiSecret string
-	client    HTTPClient
+	apiKey     string
+	apiSecret  string
+	HTTPClient *http.Client
```

Comment:
Oh OK. We only need to `gock.InterceptClient` when the client  doesn't use the default `Transport`. So I think we can remove all the `interceptClient` calls.

But still, should we leave the `HTTPClient` public?